notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p>Since we're doing this twice, it's ok to jam everything together into one cell, as long as they retain their comments.</p><p> </p><p>Also, why are we filtering this data for a SNR of &gt;1, but the other data is &gt;2? We should make them consistent or explain the need for them to be different</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>